app/main.py

```diff
@@ -1,3 +1,59 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
```

Comment:
add annotations everywhere, for return values as well, and consider that `km` can be `int` or `float`, and `other` can be `int` or `float` or even `Distance` somewhere

Reply:
> add annotations everywhere, for return values as well, and consider that `km` can be `int` or `float`, and `other` can be `int` or `float` or even `Distance` somewhere
I changed annotations for 'other' and for 'km'
And for return values all the annotations are there...